infrastructure/hearth-plugins/checkDuplicateTask.js

```diff
@@ -35,7 +35,7 @@ module.exports = (mongo, fhirResources) => {
                   'http://opencrvs.org/specs/id/death-tracking-id'
             )
             if (!id) {
-              callback(null, null)
+              return callback(null, null)
```

Comment:
This is a bug I noticed when I was (most likely) sending an invalid payload. By the looks of the code, it should stop at this point. Otherwise, an undefined property error is thrown.